### PR TITLE
Local datastore migration

### PR DESCRIPTION
**Pull request by whuman**

### CODE REVIEW

File: Server/Hambasafe.Api/Startup.cs [hunk 1]
```diff
@@ -85,6 +82,17 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
...
             loggerFactory.AddDebug();
 
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+
+                using (var serviceScope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope())
+                {
+                    serviceScope.ServiceProvider.GetService<HambasafeDataContext>().Database.Migrate();
+                    //serviceScope.ServiceProvider.GetService<ApplicationDbContext>().Database.Migrate();
```
Comment (whuman) [inline Server/Hambasafe.Api/Startup.cs]: Only create and migrate HambasafeDataContext schema for now.  ApplicationDbContext can be added when we need it.  This allows us to get up and running in your local environment relatively painlessly.

File: Server/Hambasafe.Api/appsettings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿{
   "Data": {
     "DefaultConnection": {
-      "ConnectionString": "Server=localhost\\SQLEXPRESS;Database=hsdevdb1;Trusted_Connection=True;"
+      "ConnectionString": "Server=(localdb)\\MSSQLLocalDB;Database=hsdevdb1;Trusted_Connection=True;"
```
Comment (whuman) [inline Server/Hambasafe.Api/appsettings.json]: I'm making an assumption here that everyone might not be running SQL Express on their local environment.  Visual Studio 2015 creates a localdb environment that can be used to host the dev database.

**Comment by whuman** [comment]
Not sure if we to have this Pull Request review process going forward or if you are happy with just simply merging pull requests

**Comment by belatu** [comment]
Until end of April, just go ahead and merge, as of 1 May we will start following a review process like.

Sent by Mailwise – Your emails, with style.:)
    

-------- Original Message --------

**Pull request merged**